src/main/java/techreborn/blocks/cable/BlockCable.java

```diff
@@ -145,7 +149,11 @@ public EnumBlockRenderType getRenderType(IBlockState state) {
 	@Nullable
 	@Override
 	public TileEntity createNewTileEntity(World worldIn, int meta) {
-		return new TileCable(getStateFromMeta(meta).getValue(TYPE));
+		if (Loader.isModLoaded("ic2") && PowerSystem.EnergySystem.EU.enabled.get()) {
```

Comment:
Might not be a bad idea to add a config option for this, as it could change the behavour of the cables in existing worlds.

Right now people could be using our cables to transfer RF power when IC2 is installed. I do think its out of scope and not a good idea to have a cable to works with both EU and FE, but having a config to force FE on the cables might not go amiss.